src/types/onyx/Policy.ts

```diff
@@ -591,6 +594,361 @@ type XeroConnectionConfig = OnyxCommon.OnyxValueWithOfflineFeedback<{
     errorFields?: OnyxCommon.ErrorFields;
 }>;
 
+/** Data stored about subsidiaries from NetSuite  */
+type Subsidiary = {
+    /** ID of the subsidiary */
+    internalID: string;
+
+    /** Country where subsidiary is present */
+    country: string;
+
+    /** Whether the subsidiary is an elimination subsidiary (a special type used to handle intercompany transaction) */
+    isElimination: boolean;
+
+    /** Name of the subsidiary */
+    name: string;
+};
+
+/** NetSuite bank account type values imported by Expensify */
+type AccountTypeValues = '_accountsPayable' | '_otherCurrentLiability' | '_creditCard' | '_bank' | '_otherCurrentAsset' | '_longTermLiability' | '_accountsReceivable' | '_expense';
+
+/** NetSuite Financial account (bank account, debit card, etc) */
+type NetSuiteAccount = {
+    /** GL code assigned to the financial account */
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    'GL Code': string;
+
+    /** Name of the account */
+    name: string;
+
+    /** ID assigned to the financial account in NetSuite */
+    id: string;
+
+    /** Type of the financial account */
+    type: AccountTypeValues;
+};
+
+/** NetSuite Tax account */
+type NetSuiteTaxAccount = {
+    /** Name of the tax account */
+    name: string;
+
+    /** ID assigned to the tax account in NetSuite */
+    externalID: string;
+
+    /** Country of the tax account */
+    country: string;
+};
+
+/** NetSuite Vendor Model */
+type NetSuiteVendor = {
+    /** ID of the vendor in NetSuite */
+    id: string;
+
+    /** Name of the vendor */
+    name: string;
+
+    /** E-mail of the vendor */
+    email: string;
+};
+
+/** NetSuite Invoice Item Model */
+type InvoiceItem = {
+    /** ID of the invoice item in NetSuite */
+    id: string;
+
+    /** Name of the invoice item */
+    name: string;
+};
+
+/** Data from the NetSuite accounting integration. */
+type NetSuiteConnectionData = {
+    /** Collection of the subsidiaries present in the NetSuite account */
+    subsidiaryList: Subsidiary[];
+
+    /** Collection of receivable accounts */
+    receivableList: NetSuiteAccount[];
```

Comment:
NAB, but do we know if all these keys will always exist? For now based on the sample data this looks correct.